Frontend/ui-driver/src/Screens/UploadDrivingLicenseScreen/ComponentConfig.purs

```diff
@@ -56,8 +56,8 @@ primaryButtonConfig state = let
       , margin = if imageUploadCondition then Margin 15 0 15 10 else Margin 15 0 15 30
       , cornerRadius = 6.0
       , height = V 50
-      , isClickable =  state.data.dob /= "" && DS.length state.data.driver_license_number >= 9 && (DS.toLower(state.data.driver_license_number) == DS.toLower(state.data.reEnterDriverLicenseNumber)) && state.data.dateOfIssue /= Just ""
-      , alpha = if (state.data.dob /= "" && DS.length state.data.driver_license_number >= 9) && (DS.toLower(state.data.driver_license_number) == DS.toLower(state.data.reEnterDriverLicenseNumber)) && state.data.dateOfIssue /= Just "" then 1.0 else 0.8
+      , isClickable =  state.data.dob /= "" && DS.length state.data.driver_license_number >= 9 && (DS.toLower(state.data.driver_license_number) == DS.toLower(state.data.reEnterDriverLicenseNumber) || (not state.data.cityConfig.uploadRCandDL)) && state.data.dateOfIssue /= Just ""
```

Comment:
Can we create a helper function in MobilityPrelude to handle the case-insensitive comparison and then use it here